src/components/velocity.js

```diff
@@ -0,0 +1,25 @@
+var registerComponent = require('../core/component').registerComponent;
+
+/**
+ * Velocity component.
+ *
+ * Updates object's position at each tick.
+ */
+module.exports = registerComponent('velocity', {
+  dependencies: ['position'],
```

Comment:
Oh, now that you put it that way it does make sense. I guess `getComputedAttribute` is generally what I want (to avoid initial `undefined`) then.